src/Card/CardSelect.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import Form from '../Form';
+
+const CardSelect = ({
+  className,
+  cardsData,
+  onChange,
+  selectedCardName,
+}) => {
+  const [currentRecord, setCurrentRecord] = useState(null);
+  useEffect(() => {
+    setCurrentRecord(selectedCardName);
+  }, []);
+
+  return (
+    <>
+      <Form.Group>
+        <Form.RadioSet
+          name="radio-select-card"
+          onChange={(e) => {
+            setCurrentRecord(e.target.value);
+            onChange(e.target.value);
+          }}
+          defaultValue={selectedCardName}
+          isInline
+          className="row"
+        >
+          {cardsData?.map(data => (
+            <div
+              key={data.name}
+              className={className}
+              role="group"
+              aria-label={data.title}
+            >
+              <Card className={currentRecord === data.name ? 'border border-dark' : null}>
+                <Card.Img variant="top" src={data.img} />
+                <Card.Body>
+                  <Card.Title as="h5" className="card-title mb-1">
+                    <>{data.title}</>
+                    <Form.Radio value={data.name} />
```

Comment:
Can we please add at least two subtitles here conditionally? That is the requirement for us as well where we need to show these cards.

Reply:
Done